public/documentation/_data.json

```diff
@@ -13,6 +13,9 @@
     "title": "开始",
     "origin": "getting_started"
   },
+  "crosswalk-app-tools": {
```

Comment:
Yes I believe so, and it seems that we forget to add an entry here for english version when the contents get submitted.